oex_plugin/openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -4,12 +4,128 @@
 
 from django.utils.translation import gettext_lazy as _
 
+from openedx.core.djangoapps.user_api import errors
+from openedx.core.djangoapps.user_api.errors import (
+    AccountUpdateError,
+    AccountValidationError,
+    PreferenceValidationError
+)
+from openedx.core.djangoapps.user_api.accounts.api import (
+    _get_old_language_proficiencies_if_updating,
+    _get_user_and_profile,
+    _notify_language_proficiencies_update_if_needed,
+    _send_email_change_requests_if_needed,
+    _store_old_name_if_needed,
+    _update_extended_profile_if_needed,
+    _update_preferences_if_needed,
+    _update_state_if_needed,
+    _validate_email_change,
+    _validate_name_change,
+    _validate_read_only_fields,
+    _validate_secondary_email,
+)
+from openedx.core.djangoapps.user_api.accounts.serializers import (
+    AccountLegacyProfileSerializer,
+    AccountUserSerializer,
+)
+from openedx.core.lib.api.view_utils import add_serializer_errors
 
-def get_name_validation_error(prev_func, name):
+
+def get_name_validation_error(original_func, name):
     """
     Addition validation for allowed name length.
     """
     if name and len(name) > 255:
         return _("Full name can't be longer than 255 symbols")
 
-    return prev_func(name)
+    return original_func(name)
+
+
+def update_account_settings(original_func, requesting_user, update, username=None):
+    """
+    Adding processing for extended profile fields.
+
+    Update user account information.
```

Comment:
Minor: I don't like the override description as a main doc title. It is also not clear that this is the override description.

I would rather add a separate section like it is done for the "Note" below, e.g.:
```
Override description:
    Some useful info about the override.
    Some more info...
```

Reply:
done